content/solana-nft.md

```diff
@@ -674,7 +623,7 @@ ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts sign_all \
 
 # Challenge
 
-Now it’s your turn to build something independently. Create an new Candy Machine using your own images, metadata, and configuration.
+Now it’s your turn to build something independently. Create a new Candy Machine using your own images, metadata, and configuration.
 
 You can build this from scratch or you can download the starter code here.
```

Comment:
@ZYJLiu this is the last thing needing attention... I think we can just lose this sentence altogether?  (It should be #obv that all they need to start is assets and a config file)  What do you think?